btk/obs_conditions.py

```diff
@@ -99,51 +164,69 @@ def get_psf_sky(self, psf_stamp_size):
 
 
 class CosmosCutout(Cutout):
-    def __init__(
-        self, stamp_size, pixel_scale, psf_stamp_size=41, psf_scale=None, **wcs_kwargs
-    ):
-        super(CosmosCutout, self).__init__(stamp_size, pixel_scale, **wcs_kwargs)
-        if psf_scale is None:
-            psf_scale = 0.3 * pixel_scale
+    def __init__(self,
+                 stamp_size,
+                 survey,
+                 band,
+                 psf_stamp_size=41,
+                 ):
+        """Class containing the necessary information to draw a postage stamp (PSF,
+        pixel_scale, WCS, etc.) using Cosmos' real galaxies for a given survey.
+
+        Parameters
+        ----------
+        stamp_size: int
+            size of the image cutout in pixels
+        survey: `Survey`
+            a `Survey` object that contains the meta-information for the cutout
+        psf_stamp_size: int
+            size of the psf stamp in pixels
 
-        self.psf_scale = psf_scale
+        """
+        super(CosmosCutout, self).__init__(stamp_size, survey.pixel_scale)
+        self.survey = survey
         self.psf_stamp_size = psf_stamp_size
+        self.band = band
 
     @staticmethod
     def psf_function(r):
         return galsim.Moffat(2, r)
```

Comment:
Do you think there should be support for some other PSF profile, perhaps provided by the user ? I was thinking about having the psf_function as an attribute of the cutout, allowing users to input a custom one via modified ObsConditions.

Reply:
This would be a good addition. The most flexible PSF format would be an image itself. It would be also nice to add an functionality to talk to DM stack which enable BTK to simulate Rubin images with the PSF determined from data (and also HSC images, which is actually what I want to use :))

Reply:
Thanks for the suggestion, @HironaoMiyatake please go ahead and open and issue and we can all think about this next